### DISCUSSION TITLE
LOBs and XMLs

### DISCUSSION THREAD

**Discussion by sergibielalbiol** [ismailhammounou/db2ixf#85] [Q&A] [answered]
Dear @ismailhammounou 

First of all thanks a lot for sharing this tool.

I am wondering if db2ixf can handle LOBs and XMLs column types.

Thank You!

**Comment by ismailhammounou** [comment, marked as answer]
Hi @sergibielalbiol ,

These are complex data types and it is a real challenge to add support for both in db2ixf. For now, I am not planning to support xml and lob data types but if I see  high demand then I will try to help and add support.

I still search for contributors as I have other projects I work on in my free time.

Thank you for your feedback and your understanding. 

Have a good day,
Ismail

**Reply by sergibielalbiol** [reply]
I appreciate your feedback @ismailhammounou, it is indeed complex to manage with LOBs and XMLs!

Again, thank you and very nice job!
Sergi
